mods/SizzurpMods.md

```diff
@@ -1,4 +1,4 @@
-# SizzurpMods
+# 🙂 SizzurpMods
```

Comment:
Cool, could you just add the rating to the README.md as well?

Reply:
are u blind